tree/dataframe/inc/ROOT/RDFNodes.hxx

```diff
@@ -13,6 +13,7 @@
 
 #include "ROOT/GraphNode.hxx"
 #include "ROOT/RActionBase.hxx"
+#include "ROOT/RDFAction.hxx"
```

Comment:
There is also the conflict between `RAction` for the name of the class, and `RDFAction.hxx` for the header...

Reply:
I agree this is awkward. As discussed between us and with the agreement of @Axel-Naumann , I will move all headers that are not meant to be included by users to a `RDF` subdirectory